### PR TITLE
BS-123

### PR DESCRIPTION
**Pull request by daviemukungi**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more readable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Summary
Workflows to clone ICD10 extensions repository, build the jar and add the jar to the execution classpath

## Screenshot
snomedScreenshot

## JIRA tickets
https://bahmni.atlassian.net/browse/BS-123

### CODE REVIEW

File: .github/workflows/build_publish.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ on:
     branches:
       - snomed-master
       - 'snomed-release-*'
+      - reports-issue
```
Comment (sivareddyp) [inline .github/workflows/build_publish.yml:7]: please remove this branch.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+FROM amazoncorretto:8
```
Comment (sivareddyp) [inline Dockerfile:1]: where are you using newly created file ?

File: package/docker/bahmni-reports/Dockerfile [hunk 3]
```diff
@@ -22,7 +24,10 @@ COPY target/bahmnireports.war /etc/bahmni-reports/bahmnireports.war
 RUN cd ${WAR_DIRECTORY} && jar xvf /etc/bahmni-reports/bahmnireports.war
 
 ADD https://repo.mybahmni.org/packages/build/bahmni-embedded-tomcat-8.0.42.jar /opt/bahmni-reports/lib/bahmni-embedded-tomcat.jar
+RUN cd /opt/ && git clone https://github.com/Bahmni/bahmni-reports-extension-icd10.git
+RUN cd /opt/bahmni-reports-extension-icd10 && mvn clean install
 
+RUN cp /opt/bahmni-reports-extension-icd10/target/extensions-1.0-SNAPSHOT.jar ${WAR_DIRECTORY}/WEB-INF/lib/extensions-1.0-SNAPSHOT.jar
```
Comment (sivareddyp) [inline package/docker/bahmni-reports/Dockerfile:30]: please follow `semantic versioning` for version

File: src/main/java/org/bahmni/reports/template/TSIntegrationDiagnosisLineReportTemplate.java [hunk 4]
```diff
@@ -99,7 +99,7 @@ private void enrichUsingReflection(String extensionClassStr, Collection<Map<Stri
             Constructor constructor = extensionClass.getDeclaredConstructor();
             constructor.setAccessible(true);
             ResultSetExtension extension = (ResultSetExtension) constructor.newInstance();
-            extension.enrich(collection, jasperReport);
+            extension.enrich(collection, jasperReport);//enrich
```
Comment (sivareddyp) [inline src/main/java/org/bahmni/reports/template/TSIntegrationDiagnosisLineReportTemplate.java:102]: please remove this comment

File: package/docker/bahmni-reports/Dockerfile [hunk 5]
```diff
@@ -22,7 +24,10 @@ COPY target/bahmnireports.war /etc/bahmni-reports/bahmnireports.war
 RUN cd ${WAR_DIRECTORY} && jar xvf /etc/bahmni-reports/bahmnireports.war
 
 ADD https://repo.mybahmni.org/packages/build/bahmni-embedded-tomcat-8.0.42.jar /opt/bahmni-reports/lib/bahmni-embedded-tomcat.jar
+RUN cd /opt/ && git clone https://github.com/Bahmni/bahmni-reports-extension-icd10.git
+RUN cd /opt/bahmni-reports-extension-icd10 && mvn clean install
```
Comment (sivareddyp) [inline package/docker/bahmni-reports/Dockerfile:28]: you can remove the directory `/opt/bahmni-reports-extension-icd10` after copying of jar file

**Comment by daviemukungi** [comment]
Not needed

**Pull request closed without merging**